src/HID-APIs/SurfaceDialAPI.h

```diff
@@ -49,17 +49,29 @@ class SurfaceDialAPI
   inline void end(void);
   inline void click(void);
   inline void rotate(int16_t rotation);
+	inline void position(int8_t x, int8_t y);
```

Comment:
please review your indentation settings. now it looks like it is wrongly indented.